### ISSUE TITLE
layer support

### ISSUE THREAD

**Issue by andrewzwicky** [andrewzwicky/PUBGIS#16]
Ideally, paths could be exported in a format that allows layers so that multiple paths could be superimposed on the same image (done later in GIMP, etc.)

https://www.reddit.com/r/PUBATTLEGROUNDS/comments/72xkir/pubgis_program_to_plot_your_path_during_an_entire/dnmi868/?st=j84kxcgo&sh=d481a69f

**Comment by andrewzwicky** [comment]
Yeah the coords are stored already as the processing is done, this would just be a change to https://github.com/andrewzwicky/PUBGIS/blob/bc61c6d8f4845480a52bf163d01e7359cb46e562/pubgis/match.py#L260-L279

There's a lof of potential for different output options there.

**Comment by kokarn** [comment]
Yeah, that's great!

A simple SVG path, for example, would be a great visualization.
